app/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,8 @@
     <string name="action_find">Find</string>
     <string name="text_barcode">Type the barcode :</string>
     <string name="message_in_activity_save_product_offline">Only one photo is compulsory to send a new product.</string>
+    <string name="no_offline_data">All your contributions have been sent.</string>
```

Comment:
Maybe this would sound better as
> You're all done!
Just something a little more upbeat and giving the user a sense of achievement